### PR TITLE
legacy course notifications are now expanded upon

### PR DESCRIPTION
**Pull request by Peedee2002**
- added legacy course sign
- recalled the scrapers for programs and specialisation
Screenshot 2022-04-09 192043
s

### CODE REVIEW

**Review by gtangelo — APPROVED** [review]
lgtm!

File: frontend/src/pages/TermPlanner/DraggableCourse.jsx [hunk 1]
```diff
@@ -95,9 +94,9 @@ function DraggableCourse({ code, index }) {
 	  and the course has no warning, then disable the tooltip */}
       {!isDragDisabled && (warning1 || warning2) && (
         <ReactTooltip id={code} place="bottom" className="tooltip">
-          {isLegacy ? "This course is discontinued." : 
+          {isLegacy ? "This course is discontinued. If an equivalent course is currently being offered, please pick that instead." : 
             !isUnlocked ? prereqDisplay : 
-              warningMessage != "" ? warningMessage : handbook_note}
+              warningMessage !== "" ? warningMessage : handbook_note}
```
Comment (hdphuong) [inline frontend/src/pages/TermPlanner/DraggableCourse.jsx:99]: whoaa huge respect for getting my super cooked logic here 😂  tbf, is there any other way of dealing with this not involving ternary operators?

**Review by hdphuong — APPROVED** [review]
lgtm!

File: frontend/src/pages/TermPlanner/DraggableCourse.jsx (hunk 1 above)
Comment (Peedee2002) [reply frontend/src/pages/TermPlanner/DraggableCourse.jsx:99]: Ill be honest - this is also the code i would have written lmao

**Pull request merged**